client/test/e2e/arc/views/landing-view.js

```diff
@@ -27,6 +27,66 @@ var LandingView = (function () {
       10000);
     };
 
+    this.openBuildDeployView = function() {
+      browser.driver.wait(
+        EC.presenceOf(this.buildDeployAppCommand),
+      10000);
+      this.buildDeployAppCommand.click();
+      browser.driver.wait(
+        EC.presenceOf(
+          element(by.css('input[ng-model="deploy.universal.archive"]'))
```

Comment:
technically this is a deploy oriented element.  It should be switched to a build element (maybe the build button)

the build and deploy views may be decoupled in the UI in the future.